tests/acceptance/helpers/webdavHelper.js

```diff
@@ -217,7 +217,7 @@ exports.getSkeletonFile = function (filename) {
     })
 }
 
-exports.uploadFileWithcontent = function (user, content, filename) {
+exports.uploadFileWithContent = function (user, content, filename) {
```

Comment:
The function name is changed but it is not updated anywhere. Where is this function called from ?